sympy/integrals/__init__.py

```diff
@@ -9,7 +9,11 @@
     >>> integrate(sin(x),x)
     -cos(x)
 """
+from .deltafunctions import deltaintegrate
+from .heurisch import heurisch, heurisch_wrapper
 from .integrals import integrate, Integral, line_integrate
+from .manualintegrate import manualintegrate, integral_steps
+from .rationaltools import ratint
```

Comment:
On Mon, Mar 09, 2015 at 05:30:25AM -0700, Joachim Durchholz wrote:
>    If the API documentation is wrong, they should be removed there (and
>    here).

If the api docs is wrong - you can fix one.  Not with adding new exports.

As I can see in the stable version, the docs examples are
correct, not wrong:
http://docs.sympy.org/latest/modules/integrals/integrals.html#api-reference
e.g. they do "from sympy.integrals.deltafunctions import deltaintegrate"...